sync/clients/sync.py

```diff
@@ -13,7 +13,8 @@ class SyncAuth(httpx.Auth):
     requires_response_body = True
 
     def __init__(self, api_url: str, api_key: APIKey):
-        self.auth_url = f"{api_url}/v1/auth/token"
+        # self.auth_url = f"{api_url}/v1/auth/token"
+        self.auth_url = "https://dev-api.synccomputing.com/v1/auth/token"
```

Comment:
This can be achieved by setting `api_url` in ~/.sync/config

Reply:
Ah, yeah, I did not mean to commit this 😅  I had my config pointed to local but wanted the auth call to point to dev, since fetching a token from my local address was causing me issues for some reason